engine/runtime/src/bench/scala/org/enso/interpreter/bench/fixtures/semantic/AtomFixtures.scala

```diff
@@ -28,18 +28,18 @@ class AtomFixtures extends DefaultInterpreterRunner {
   val generateListQualifiedCode =
     """from Standard.Base.Data.List import all
       |
-      |main = length ->
-      |    generator = acc -> i -> if i == 0 then acc else @Tail_Call generator (List.cons i acc) (i - 1)
+      |main self = length ->
+      |    generator = acc -> i -> if i == 0 then acc else @Tail_Call generator (Cons i acc) (i - 1)
```

Comment:
Should be List.Cons – the point of this benchmark is to verify there's no overhead in the qualified form.